cornflow/endpoints/execution.py

```diff
@@ -63,8 +63,9 @@ def post(self):
         self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
         result = self.post_list(request)
 
-        # TODO: check result[1] before continuing! if not 20X, return.
-        if result[1] >= 300:
+        not_run = request.args.get('run', 'no') == 'no'
```

Comment:
why a string and not a boolean? I think it should be a 1, 0 in the request args. And called `/execution/?run=0 or /execution/?run=1` with default=1.

Reply:
It can be a number in the url but the url parser is going to take it as string anyway, but i can change it to 0 and 1

Reply:
ok i thought that in fact it had to be added to the endpoint in app.py in line https://github.com/baobabsoluciones/corn/blob/e5a780c5b6024f3cdae2d0c592376c99986eafe3/cornflow/app.py#L34 and there we could force it to be a binary. Right? Although I guess we would need to split the "post" from the "get" in different endpoints in order to only ask for an optional argument only in the post. We can leave it like this for now knowing it should be "documented" somewhere.